src/runtime/server.ts

```diff
@@ -8,6 +8,9 @@ import express, {
 } from 'express';
 import userAgentMiddleware from 'express-useragent';
 import nocache from 'nocache';
+import {printRouteInfo} from '../printers/start';
+import watch from 'node-watch';
```

Comment:
Could you try this library instead? https://www.npmjs.com/package/chokidar? It seems like this is a defacto standard. It's being used by libraries like nodemon, babel and browser-sync

Reply:
That's a much better library to use for this use case - I didn't realize that babel was using that!  I'll definitely make that change.